lib/containers/common.pm

```diff
@@ -295,6 +295,9 @@ sub check_containers_connectivity {
     record_info "connectivity", "Checking that containers can connect to the host, to each other and outside of the host";
     my $container_name = 'sut_container';
 
+    record_info 'https://sd.suse.com/servicedesk/customer/portal/1/SD-135489';
```

Comment:
You are missing the first argument in `record_info`. It would be better to use the link as the second/description parameter and set a title too

Reply:
True, fixed.